cmd/agent/app/reporter/grpc/flags.go

```diff
@@ -41,11 +41,6 @@ var tlsFlagsConfig = tlscfg.ClientFlagsConfig{
 func AddFlags(flags *flag.FlagSet) {
 	flags.Uint(retry, defaultMaxRetry, "Sets the maximum number of retries for a call")
 	flags.Int(discoveryMinPeers, 3, "Max number of collectors to which the agent will try to connect at any given time")
-	AddOTELFlags(flags)
-}
-
-// AddOTELFlags adds flags that are exposed by OTEL collector
-func AddOTELFlags(flags *flag.FlagSet) {
 	flags.String(collectorHostPort, "", "Comma-separated string representing host:port of a static list of collectors to connect to directly")
```

Comment:
I wonder why these were considered specific to OTEL in the first place

Reply:
Not all flags were added to OTEL distros in other words some flags made sense only for legacy Jaeger.